lib/auth/init.go

```diff
@@ -551,6 +551,20 @@ func shouldInitReplaceResourceWithOrigin(stored, candidate types.ResourceWithOri
 	return false, nil
 }
 
+// migrationStart marks the migration as active.
+// It should be called when a migration starts.
+func migrationStart(ctx context.Context, migrationName string) {
+	log.Debugf("Migrations: %q migration started.", migrationName)
+	migrations.WithLabelValues(migrationName).Set(float64(1))
```

Comment:
I don't think you need this typecast

Reply:
Thanks! Removed in 3083c9b8170cc8e8e319d2d6d896c59da45789bd.